29787 text wrap (#30435)
mdn/content: files/en-us/mozilla/firefox/releases/121/index.md

### CSS

> - The {{CSSXRef("text-wrap")}} property has now been added with support for the values `balance` and `stable`. The `balance` value is used for short blocks of content, such as headings, and will make sure that when spread over multiple lines the content is balanced and easy to read. The `stable` value makes sure that editable content does not reflow will the user is editing ([Firefox bug 1731541](https://bugzil.la/1731541)).

Reviewer: - subjective clause position improvement
- "added" -> "updated" (since `text-wrap` as a property has already been present)
- "will" -> "while"
- added "it" token

```suggestion
- The {{CSSXRef("text-wrap")}} property has been updated with support for the values `balance` and `stable`. The `balance` value is used for short blocks of content, such as headings, and will make sure that the content is balanced and easy to read when spread over multiple lines. The `stable` value makes sure that editable content does not reflow while the user is editing it ([Firefox bug 1731541](https://bugzil.la/1731541)).
```
